### PR TITLE
added lightbubls that player can toggle with E in close proximity

### PR DESCRIPTION
**Pull request by dadisigursveinn**

### CODE REVIEW

**Comment by dadisigursveinn** [comment]
[Player can change light bulbs](https://trello.com/c/KzWGn7oK/54-player-can-change-light-bulbs)

File: Assets/Prefabs/LightBulb.prefab [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 100100000}
+  serializedVersion: 5
+  m_Component:
+  - component: {fileID: 4178140233142332}
+  - component: {fileID: 212816048056893942}
+  - component: {fileID: 114346396841669666}
+  m_Layer: 0
+  m_Name: LightBulb
+  m_TagString: Untagged
```
Comment (hallur) [inline Assets/Prefabs/LightBulb.prefab]: Shouldn't the tag string be set?

File: Assets/Scripts/LightBulbController.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LightBulbController : MonoBehaviour
+{
+
+    public bool lightOn = false;
```
Comment (hallur) [inline Assets/Scripts/LightBulbController.cs]: Class variables should be initialized in Prefab.prefab according to previous lecture if I'm not mistaken

File: Assets/Prefabs/LightBulb.prefab (hunk 1 above)
Comment (dadisigursveinn) [reply Assets/Prefabs/LightBulb.prefab]: k

**Review by hallur — CHANGES_REQUESTED** [review]

File: Assets/Scripts/LightBulbController.cs (hunk 2 above)
Comment (dadisigursveinn) [reply Assets/Scripts/LightBulbController.cs]: not necessarily but it has been changed

**Review by hallur — APPROVED** [review]

**Pull request merged**